exercises/diffie-hellman/src/test/kotlin/DiffieHellmanTest.kt

```diff
@@ -0,0 +1,79 @@
+import org.junit.Before
+import org.junit.Ignore
+import org.junit.Test
+import java.math.BigInteger
+import kotlin.test.assertEquals
+import kotlin.test.assertNotEquals
+import kotlin.test.assertTrue
+
+class DiffieHellmanTest {
+
+    private lateinit var diffieHellman: DiffieHellman
+
+    @Before
+    fun setUp() {
+        diffieHellman = DiffieHellman()
+    }
+
+    @Test
+    fun testPrivateKeyIsInRange() {
+        val prime = 23.toBigInteger()
+        (0..9).map { diffieHellman.privateKey(prime) }.forEach {
+            assertTrue(it >= BigInteger.ONE)
+            assertTrue(it < prime)
+        }
+    }
+
+    // Note: due to the nature of randomness, there is always a chance that this test fails
+    // Be sure to check the actual generated values
```

Comment:
Can we form this note as a regular kdoc-comment?

Reply:
I guess it's better to make a follow-up issue to fix this later. Tests should introduce unexpected behavior never. :) But this more a question to specification.